static/app/views/performance/newTraceDetails/traceModels/traceTree.tsx

```diff
@@ -130,7 +130,15 @@ export declare namespace TraceTree {
   type TraceError = TraceErrorType;
   type TracePerformanceIssue = TracePerformanceIssueType;
   type Profile = {profile_id: string} | {profiler_id: string};
+  type Project = {
+    id: number;
+    slug: string;
+  };
   type Root = null;
+  type Title = {
```

Comment:
I dont think we need this here since title isnt something that exists on the tree anymore.

Reply:
Correct, good catch